packages/graphql/tests/tck/tck-test-files/interface-relationships/create/connect.test.ts

```diff
@@ -94,52 +94,52 @@ describe("Interface Relationships - Create connect", () => {
         });
 
         expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
-            "CALL {
-            CREATE (this0:Actor)
-            SET this0.name = $this0_name
-            WITH this0
-            CALL {
-            	WITH this0
-            	OPTIONAL MATCH (this0_actedIn_connect0_node:Movie)
-            	WHERE this0_actedIn_connect0_node.title STARTS WITH $this0_actedIn_connect0_node_param0
-            	FOREACH(_ IN CASE WHEN this0 IS NULL THEN [] ELSE [1] END |
-            		FOREACH(_ IN CASE WHEN this0_actedIn_connect0_node IS NULL THEN [] ELSE [1] END |
-            			MERGE (this0)-[this0_actedIn_connect0_relationship:ACTED_IN]->(this0_actedIn_connect0_node)
-            SET this0_actedIn_connect0_relationship.screenTime = $this0_actedIn_connect0_relationship_screenTime
-            		)
-            	)
-            	RETURN count(*) AS _
-            UNION
-            	WITH this0
-            	OPTIONAL MATCH (this0_actedIn_connect0_node:Series)
-            	WHERE this0_actedIn_connect0_node.title STARTS WITH $this0_actedIn_connect0_node_param0
-            	FOREACH(_ IN CASE WHEN this0 IS NULL THEN [] ELSE [1] END |
-            		FOREACH(_ IN CASE WHEN this0_actedIn_connect0_node IS NULL THEN [] ELSE [1] END |
-            			MERGE (this0)-[this0_actedIn_connect0_relationship:ACTED_IN]->(this0_actedIn_connect0_node)
-            SET this0_actedIn_connect0_relationship.screenTime = $this0_actedIn_connect0_relationship_screenTime
-            		)
-            	)
-            	RETURN count(*) AS _
-            }
-            RETURN this0
-            }
-            WITH this0
-            CALL {
-            WITH this0
-            CALL {
-                WITH this0
-                MATCH (this0)-[:ACTED_IN]->(this0_Movie:Movie)
-                RETURN { __resolveType: \\"Movie\\", runtime: this0_Movie.runtime, title: this0_Movie.title } AS actedIn
-                UNION
-                WITH this0
-                MATCH (this0)-[:ACTED_IN]->(this0_Series:Series)
-                RETURN { __resolveType: \\"Series\\", episodes: this0_Series.episodes, title: this0_Series.title } AS actedIn
-            }
-            RETURN collect(actedIn) AS actedIn
-            }
-            RETURN [
-            this0 { .name, actedIn: actedIn }] AS data"
-        `);
+"CALL {
```

Comment:
question: How come the indentation is suddenly changed?

Reply:
This is a common problem with jest snapshots when the version of jest is not up to date (not sure why). I'll fix these in the PR